Update tutorials landing page (#27)
ripple/xrpl-org-dev-portal: docs/tutorials/index.mdx

The XRP Ledger tutorials walk you through the steps to learn and get started with the XRP Ledger and to use the ledger for advanced use cases.

## Beginner Tutorials

<XRPLCardDeck cols="1/3">
  <DeckCard title="Quickstart" to="./quickstart/index.md">
    This quickstart tutorial walks you through a test harness interface to try out the XRP Ledger.
  </DeckCard>
</XRPLCardDeck>

## Getting Started with SDKs

These tutorials walk you through the basics of building a very simple XRP Ledger-connected application using SDKs.

<XRPLCardDeck cols="1/4">
  <DeckCard title="Python" to="https://xrpl.org/get-started-using-python.html" external>
    Using xrpl.py, a pure Python library. 
  </DeckCard>
  <DeckCard title="Java" to="https://xrpl.org/get-started-using-java.html" external>
    Using xrpl4j, a pure Java library. 
  </DeckCard>
  <DeckCard title="JavaScript" to="https://xrpl.org/get-started-using-javascript.html" external>
    Using the xrpl.js client library. 
  </DeckCard>
  <DeckCard title="HTTP & Websocket APIs" to="https://xrpl.org/get-started-using-http-websocket-apis.html" external>

>     Access the XRP Ledger directly through the APIs of its core server software, rippled.

Reviewer: I suggest leaving off the name of the core server as future-proofing since you can also use (almost) the same API via Clio and it's possible `rippled` may be renamed `xrpld` in the future.

```suggestion
    Access the XRP Ledger directly through the APIs of its core server.
```

  </DeckCard>
</XRPLCardDeck>

## Tutorials to Manage the XRP Ledger Servers

<XRPLCardDeck cols="1/4">
  <DeckCard title="Install rippled Server" to="https://xrpl.org/install-rippled.html" external>

>     These tutorials describe the steps to install and update the rippled server on supported platforms.

Reviewer: I'd say "on supported platforms" is generally understood, and in fact the build instructions can be adapted for unsupported platforms so it's maybe not even strictly needed. Plus, we can avoid using the (possibly changing and possibly confusing) `rippled` name.

```suggestion
    These tutorials describe the steps to install and update the core XRP Ledger server.
```

  </DeckCard>
  <DeckCard title="Configure rippled Server" to="https://xrpl.org/configure-rippled.html" external>

>     These tutorials describe the steps to customize your rippled server configuration.

Reviewer: In this case I don't know if we can really avoid the `rippled` name but you can use HTML in this context to put it in its appropriate code font.

```suggestion
    These tutorials describe the steps to customize your <code>rippled</code> server configuration.
```

  </DeckCard>
  <DeckCard title="Run rippled in Standalone Mode" to="https://xrpl.org/use-stand-alone-mode.html" external>

>     These tutorials describe the steps to run rippled in stand-alone mode and test features with a full network.

Reviewer: Aside from putting `rippled` in code font, looks like you typed the opposite of what you meant for stand-alone mode's description.

```suggestion
    These tutorials describe the steps to run <code>rippled</code> in stand-alone mode and test features without a full network.
```

  </DeckCard>
  <DeckCard title="Install Clio Server " to="https://xrpl.org/install-clio-on-ubuntu.html">

>     This tutorials describes the steps to install the Clio server on Ubuntu platform.  

Reviewer: ```suggestion
    This tutorial describes the steps to install the Clio server on Ubuntu platform.  
```
